k8s/lib.sh

```diff
@@ -43,6 +43,12 @@ k8s::common::is_strict() {
   fi
 }
 
+k8s::common::resources() {
+  mkdir -p "$SNAP_COMMON/etc/"
+  cp -r "$SNAP/etc/templates" "$SNAP_COMMON/etc/"
+  cp -r "$SNAP/etc/configurations" "$SNAP_COMMON/etc/"
```

Comment:
Under `configurations` could we have a disa-stig directory. That way we could then extend this approach to CIS hardening too.